### PR TITLE
Add `GET /gov/members` endpoint

### LINKED ISSUE #3615: Add /gov/members
Add a `/gov/members` endpoint that exposes a map of member ids to member info, including cert, status and additional data where applicable.

This is to make it easier for members to stage a recovery, particular when the status of recent key rotation is uncertain.

### PR DESCRIPTION
**Pull request by eddyashton**
Resolves #3615.

Sample response:
```
$ curl -k https://127.0.0.1:8000/gov/members | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1272  100  1272    0     0   124k      0 --:--:-- --:--:-- --:--:--  124k
{
  "ae5b09279cdbec24d0e94b7a4cf2f2654da1a101be8a299c8da3179bdf370bbc": {
    "cert": "-----BEGIN CERTIFICATE-----\nMIIBtjCCATygAwIBAgIUOq1HvRmQIXLu9gBMtF6MKxQXHM0wCgYIKoZIzj0EAwMw\nEjEQMA4GA1UEAwwHbWVtYmVyMDAeFw0yMjAzMDExMTQzMThaFw0yMzAzMDExMTQz\nMThaMBIxEDAOBgNVBAMMB21lbWJlcjAwdjAQBgcqhkjOPQIBBgUrgQQAIgNiAARF\nnMUOQ3/Nq8PNGfSosV4JujMDBvcfBnCXYoNs6S/i6Hre35COiuR48kok/QVX16bJ\naioX2wVwtdxwBQUwVBChUBLXNeRegGphw+8s/s3qSMTWwRb3Kte6JHkmK/++hyWj\nUzBRMB0GA1UdDgQWBBR0/SjlakLHzEcwANCaUkRkT8vvoTAfBgNVHSMEGDAWgBR0\n/SjlakLHzEcwANCaUkRkT8vvoTAPBgNVHRMBAf8EBTADAQH/MAoGCCqGSM49BAMD\nA2gAMGUCMCm5ddT4yXRJcbqvxQvKD4n4C/tgTO5Vi5VL+jc1/ZdWV2VVYM95XkHX\ne9pWTcWfTwIxAPPNfh0Khw5mYGOQgX06agJdfzKq4mnLvwII6to7c6gSIxmxpuHQ\n2MEqMInuhs0QrA==\n-----END CERTIFICATE-----\n",
    "member_data": null,
    "public_encryption_key": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0sh2kHh6DDmfgoMrmmy7\nRUfsIodwr0tMrdQ2qddBJCUi5+4RwwdlPny5rq4d+O8PucX1fogt97sABoh5SN+P\niaD5OF0Er+7j2d9nKhECRQbKipdCLgi4dRfByZAfEgqPQRe+txxYm49tQAYjoIqj\n/WoMiq2VdP878RXraJr7T1rVVEkpolkx6opo3JDVgPrQ3SH6i5f3ya1dFvCjoHT9\ns72D/0iGIqITgb8POg99QVZJKTbnh0yDwK0vz42mjzepK1n198wfN/aTJkv6WyZ7\nK4wP4slVnADJEES3npQSnEICPV/hb9RoRXgQuxx3PIqAfHA+OK8JJ+czIq0/R51K\nSQIDAQAB\n-----END PUBLIC KEY-----\n",
    "status": "Active"
  }
}
```

Note that this aims to be a clear summary, to avoid confusion. So for instance it always includes a `member_data` field, potentially containing `null`, rather than omitting the field for brevity.

### CODE REVIEW

**Review by wintersteiger — APPROVED** [review]
LGTM!

File: tests/governance.py [hunk 1]
```diff
@@ -212,6 +212,37 @@ def test_member_data(network, args):
     return network
 
 
+@reqs.description("Check /gov/members endpoint")
+def test_all_members(network, args):
+    primary, _ = network.find_primary()
+
+    with primary.client() as c:
```
Comment (achamayou) [inline tests/governance.py:219]: Can we check that this is accessible in recover/partOfPublicNetwork mode?

Comment (eddyashton) [reply tests/governance.py:219]: I've manually confirmed it's accessible. I don't see an obvious way to fit that into our existing tests. Could add a new test case which does a recovery, or pass a list of functions (additional test cases) to run on the mid-recovery network to `test_recover_service`... I'll take a look.

Comment (eddyashton) [reply tests/governance.py]: Now done.

**Pull request merged**